File: TEMA1/python/maxmin.py
def maxmin(a,b,c,mayor=0,menor=0):
    if a>=b and a>=c: mayor = a
    elif b>=a and b>=c: mayor = b
    else: mayor = c
    if a<=b and a<=c: menor = a
    elif b<=a and b<=c: menor = b
    else: menor = c
    return f'Mayor:{mayor} Menor:{menor}'

a=int(input("Introduce el primer numero: "))
b=int(input("Introduce el segundo numero: "))
c=int(input("Introduce el tercer numero: "))
print(maxmin(a,b,c))

File: TEMA1/typescript/maxmin.ts
import readline from "node:readline/promises";
import { stdin as input, stdout as output } from "node:process";

const rl = readline.createInterface({ input, output });

function maxmin(a: number, b: number, c: number, mayor: number = 0, menor: number = 0): string {
    if (a >= b && a >= c) { mayor = a;
    } else if (b >= a && b >= c) { mayor = b;
    } else { mayor = c;
    } if (a <= b && a <= c) { menor = a;
    } else if (b <= a && b <= c) { menor = b;
    } else { menor = c;
    }
    return `Mayor:${mayor} Menor:${menor}`;
}

const a = Number(await rl.question("Introduzca el primer numero: "));
const b = Number(await rl.question("Introduzca el segundo numero: "));
const c = Number(await rl.question("Introduzca el tercer numero: "));
console.log(maxmin(a,b,c));
rl.close();